src/pybind/mgr/volumes/module.py

```diff
@@ -70,10 +70,12 @@ class Module(orchestrator.OrchestratorClientMixin, MgrModule):
                    'name=pool_layout,type=CephString,req=false '
                    'name=uid,type=CephInt,req=false '
                    'name=gid,type=CephInt,req=false '
-                   'name=mode,type=CephString,req=false ',
+                   'name=mode,type=CephString,req=false '
+                   'name=isolate_namespace,type=CephBool,req=false ',
```

Comment:
'namespace_isolated' maybe more intuitive?

Reply:
I don't have strong opinion here. I thought verb suited better. I could change this.

Reply:
Done